metagraph/CMakeLists.txt

```diff
@@ -319,6 +323,8 @@ IF(APPLE)
     set(THREADS_PREFER_PTHREAD_FLAG ON)
 ENDIF()
 add_subdirectory(external-libraries/spdlog)
+add_subdirectory(external-libraries/sshash SYSTEM)
+target_compile_options(test_alphabet PRIVATE -Wno-strict-aliasing)
```

Comment:
Is `-Wno-strict-aliasing` still needed here? I think it should've been fixed by https://github.com/ratschlab/sshash/pull/3/commits/889fe5c1cbef86f877d2413714ea5e899c26f328.

Reply:
correct, this isn't needed anymore

Reply:
```suggestion
```